<robot name="simple_table">
  <link name="my_table">
    <inertial>
      <origin xyz="0 0 0.0145"/>
      <mass value="10" />
      <inertia  ixx="0.0001" ixy="0.0"  ixz="0.0"  iyy="0.0001"  iyz="0.0"  izz="0.0001" />
    </inertial>
    <visual>
      <origin xyz="0 0 0.215"/>
      <geometry>
        <box size="0.47 0.46 0.43"/>
      </geometry>
    </visual>
    <surface>
        <friction>
          <ode>
            <mu>0.001</mu>
            <mu2>0.001</mu2>
          </ode>
        </friction>
    </surface>
    <collision>
      <origin xyz="0 0 0.215"/>
      <geometry>
        <box size="0.47 0.46 0.43"/>
      </geometry>
    </collision>
  </link>
  <gazebo reference="my_table">
    <material>Gazebo/Wood</material>
  </gazebo>
  <gazebo>
    <static>true</static>
  </gazebo>
</robot>